services/apollo/healthcheck.js

```diff
@@ -0,0 +1,34 @@
+var http = require("http");
```

Comment:
jw how is this file related to the image changes?

Reply:
Whoops! That's not supposed to be in here. I'll remove this :)

Reply:
I was exploring using node for a healthcheck instead of curl since it's less of a security risk

Reply:
nice - glad i asked haha